integration-cli/requirements.go

```diff
@@ -29,6 +29,15 @@ var (
 		func() bool { return daemonPlatform == "windows" },
 		"Test requires a Windows daemon",
 	}
+	WindowsDaemonSupportsVolumes = testRequirement{
+		func() bool {
+			if windowsDaemonKV == 0 {
+				panic("windowsDaemonKV is not set")
```

Comment:
@jhowardmsft I think we should double check the OOS version here, just as a precaution to avoid that panic if the OOS is not windows.